src/Symfony/Component/DependencyInjection/ResettableContainerInterface.php

```diff
@@ -19,6 +19,8 @@
  * not needed anymore.
  *
  * @author Christophe Coevoet <stof@notk.org>
+ *
+ * @deprecated since Symfony 4.2, use `ResetInterface` instead.
```

Comment:
should be the FQDN here IMHO

Reply:
updated (and added to the CHANGELOG file)